core/src/json_schema.rs

```diff
@@ -263,7 +263,7 @@ pub mod simplified {
                     bson_type: None,
                     properties: None,
                     required: None,
-                    additional_properties: None,
+                    additional_properties: None | Some(false),
```

Comment:
Wait... when did they add non-top level |?!!! This is awesome and would simplify so much of my code. This definitely wasn't in rust 1.0.

Reply:
Cool!  This looks like the version that added the feature: 
https://blog.rust-lang.org/2021/06/17/Rust-1.53.0.html#or-patterns